systemtest/src/main/java/io/strimzi/systemtest/metrics/MetricsCollector.java

```diff
@@ -60,7 +60,7 @@ public static class Builder {
         private ComponentType componentType;
         private String componentName;
         private int metricsPort;
-        private String metricsPath;
+        private String metricsPath = "/metrics";
```

Comment:
So this is a default path now?
I'm wondering if this method is still needed: 

```
private String getDefaultMetricsPathForComponent() {
        switch (this.componentType) {
            case KafkaExporter:
            case UserOperator:
            case TopicOperator:
            case ClusterOperator:
            case KafkaBridge:
                return "/metrics";
            default:
                return "";
        }
}
```

Reply:
I removed it and will rerun the regression again to make sure it didn't break anything.

BTW: The right path was always `metrics/`, although the older version of the JMX Exporter had a redirect that allowed the STs to work.